server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/agents/agents_widget.js.msx

```diff
@@ -233,6 +233,7 @@ define([
 
       return (
         <div onclick={ctrl.hideDropDownsAndMessage.bind(ctrl)}>
+          <div class="extra-div-to-replace-sticky-header-height" style={!s.isBlank(message) ? "height:284px;" : ''}></div>
```

Comment:
Do not use styles, add or remove classes, 
move this method to controller.

```extra-div-to-replace-sticky-header-height``` is too specific call it ```agent-header-spacer```